customTests/custom-test-setup.sh

```diff
@@ -95,10 +95,22 @@ else
 			wget https://www.cs.virginia.edu/stream/FTP/Code/stream.c
 		fi
 
+		SKU=$( curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2021-01-01&format=text")
+		if echo $SKU | grep "HX176rs"; then
```

Comment:
Will these always have this same capitalization pattern?

Reply:
thats a good point we faced an issue like this before in azhpc images. Good catch!!!!!

Reply:
addressed